src/components/api-address-form/ApiAddressForm.js

```diff
@@ -21,8 +21,8 @@ const ApiAddressForm = ({ t, doUpdateIpfsApiAddress, ipfsApiAddress = '' }) => {
 
   return (
     <form onSubmit={onSubmit}>
-      <label htmlFor='api-address' className='db f7 mb2 ttu tracked charcoal pl1'>{t('apiAddressForm.apiLabel')}</label>
       <input id='api-address'
+        aria-labelledby={t('apiAddressForm.apiLabel')}
```

Comment:
This shouldn't work, since this attribute required a ID, not the text. I think what you're looking for is: `aria-label={t('apiAddressForm.apiLabel')}`

`aria-label` allows you to provide screen-reader text
vs
`aria-labeledby` allows you to indicate an html element that labels the element you're specifying

Reply:
Thank you! Fixed in https://github.com/ipfs-shipyard/ipfs-webui/pull/1577/commits/b01f2e01338ddaa5cb436a42d7cb2108da8ebddd.